src/client/components/NewsletterCard.tsx

```diff
@@ -114,45 +110,105 @@ const checkBoxBackgroundColorBugFix = css`
   z-index: 0 !important;
 `;
 
-const clockSVG = (
-  <svg
-    css={{ fill: brand[400] }}
-    width="15px"
-    height="15px"
-    viewBox="0 0 11 11"
-  >
-    <path d="M5.4 0C2.4 0 0 2.4 0 5.4s2.4 5.4 5.4 5.4 5.4-2.4 5.4-5.4S8.4 0 5.4 0zm3 6.8H4.7V1.7h.7L6 5.4l2.4.6v.8z" />
-  </svg>
-);
+export const iconStyles = css`
+  width: fit-content;
+  border-radius: 100%;
+  padding: 2px;
+  svg {
+    display: block;
+    fill: ${neutral[100]};
+    height: 22px;
+    width: 22px;
+  }
+  margin-right: ${space[1]}px;
+`;
+
+const descriptionBlock = css`
+  ${from.tablet} {
+    display: flex;
+    border-top: 1px solid ${neutral[86]};
+    border-bottom: 1px solid ${neutral[86]};
+  }
+`;
+
+const infoGroup = css`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+
+  ${from.tablet} {
+    padding-left: ${space[5]}px;
+  }
+`;
+
+// IE11 doesn't know that block-level elements shouldn't overflow their parent
+const ieFlexOverflowFix = css`
+  width: 100%;
+`;
+
+const frequencyStyles = css`
+  display: flex;
+  align-items: center;
+  padding: ${space[2]}px 0;
+`;
+
+const idColor: (arg0: string) => string | undefined = (id) => {
```

Comment:
was slightly confused by the type definition for this method, think this suggestion is clearer?

```suggestion
const idColor = (id: string): string => {
```

Reply:
as i don't think there is a scenario where `undefined` is returned, and was getting confused by the whole `arg0`